### PR TITLE
Do not try to import data: tel: mailto: something: URLs

### PR DESCRIPTION
**Pull request by oliger**
Parcel is trying to "import" data URLs. This PR ignore URLs when they start with `data:`.

### CODE REVIEW

**Comment by oliger** [comment]
So it seems that this PR fixes #43... And that it was already taken by @ssuman... Feel free to close.

**Comment by oliger** [comment]
Found out that Parcel needs to ignore all `something:` URLs. I made one more commit to accept URLs that start with `mailto:`, `tel:`.

File: src/utils/is-url.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 const isURL = require('is-url');
 
 module.exports = function (url) {
-  return isURL(url) || /^#/.test(url);
+  return isURL(url) || /^#/.test(url) || /^[a-z]*\:/i.test(url);
```
Comment (thejameskyle) [inline src/utils/is-url.js]: Could we hoist these regexes into constants and add descriptions?

Comment (davidnagli) [reply src/utils/is-url.js]: +1

**Comment by oliger** [comment]
Yep 👍. Try to do this tonight. 

Envoyé de mon iPhone

File: src/utils/is-url.js (hunk 1 above)
Comment (oliger) [reply src/utils/is-url.js]: @thejameskyle Done

**Comment by devongovett** [comment]
Thanks!

**Pull request merged**